lib/bundles/inspec-compliance/api/login.rb

```diff
@@ -0,0 +1,134 @@
+# encodeing: utf-8
+# author: Christoph Hartmann
+# author: Dominik Ricter
+# author: Jerry Aldrich
+
+module Login
+  def login(options)
+    options['server_type'] = Compliance::API.determine_server_type(options['server'], options['insecure'])
+
+    return Login::ComplianceServer.login(options) if options['server_type'] == 'compliance'
+    verify_thor_options(options)
+
+    options['url'] = options['server'] + '/compliance'
+    token = options['dctoken'] || options['token']
+    puts store_access_token(options, token)
+  end
+
+  private
+
+  def store_access_token(options, token)
+    token_info = { type: 'dctoken', msg: 'data colletor token' } if options['dctoken']
```

Comment:
Will you ever not have either a dctoken or a token? What happens if token_info is not set?

I'd recommend changing it to something like:

```ruby
token_info = if options['dctoken']
               { # dctoken stuff }
             else
               { # non-dctoken stuff }
             end
```

Reply:
I agree. Fixed.